src/styles.scss

```diff
@@ -21,379 +21,959 @@ html {
   overflow-y: scroll;
   overscroll-behavior-y: none;
 
-   // removes 300ms click delay on mobile browsers
-   touch-action: manipulation;
+  // removes 300ms click delay on mobile browsers
+  touch-action: manipulation;
```

Comment:
Please revert whitespace changes here - it's hard to tell what has really changed.

Reply:
ah, will fix this asap. Thanks